### PR TITLE
WIP [Refactoring] Up code quality

### PR DESCRIPTION
**Pull request by ovr**

### CODE REVIEW

File: redis-hub.go [hunk 1]
```diff
@@ -24,14 +19,10 @@ type RedisHub struct {
 	clientsToChannelsLock sync.Mutex
 }
 
-const (
-	redisSleep time.Duration = 125 * time.Millisecond
-)
-
-func NewRedisHub(client *redis.Client) *RedisHub {
+func NewRedisHub(client *redis.Client) HubInterface {
 	pubSub := client.Subscribe("controller")
```
Comment (tombouctou) [inline redis-hub.go]: can we make this configurable value?

Comment (ovr) [inline redis-hub.go]: It's a fake channel to get pubSub structure ;) Hack :3

File: server.go [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+	}
+
+	db.LogMode(config.DB.ShowLog)
+	db.DB().SetMaxIdleConns(config.DB.MaxIdleConnections)
+	db.DB().SetMaxOpenConns(config.DB.MaxOpenConnections)
+
+	server := &Server{
+		clients: map[*Client]bool{},
+		httpServer: &http.Server{
+			Addr:           ":8484",
```
Comment (tombouctou) [inline server.go]: Addr/Port should be configurable, imo

Comment (ovr) [inline server.go]: I will extract more variables to configuration 👍

**Pull request merged**